### DISCUSSION TITLE
Not writing to database in production

### DISCUSSION THREAD

**Discussion by arthurmeb** [supabase/supabase#20190] [Questions]
I have a Nuxt 3 app hosted on  Cloudflare pages. When testing in localhost, everything writes to my Supabase db correctly, but in production it doesn't. 
It used to work fine until recently, so I know it isn't my env variables. 
I can't check server logs on Cloudflare so I have really no way of investigating this issue. Any ideas?

**Comment by ThatXliner** [comment]
You should probably check out [RLS](https://supabase.com/docs/guides/auth/row-level-security). It is likely that your local database doesn’t have RLS but your production database has RLS on with no policies to allow for any access

**Reply by arthurmeb** [reply]
Turns out the issue wasn't this- Cloudflare was throwing an error earlier in the code which would cause the error to not be written to the database.
Explains why it would work on locally but not in production.
Thanks!
